migrations/dlu/12_modular_build_ugc.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS ugc_modular_build (
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	ugc_id BIGINT NOT NULL,
+    rocket_ldf VARCHAR(60) NOT NULL
```

Comment:
Rocket ldf? Isn't that supposed to work for cars too. Maybe some better generic name

Reply:
I think you can figure out why I named it rocket ldf.